contracts/common/Initializable.sol

```diff
@@ -40,4 +40,31 @@ contract Initializable is AppStorage {
     function getBlockNumber() internal view returns (uint256) {
         return block.number;
     }
+
+    /**
+    * @dev Returns the current block number, converted to uint64.
+    *      Using a function rather than `block.number` allows us to easily mock the block number in
+    *      tests.
+    */
+    function getBlockNumber64() internal view returns (uint64) {
```

Comment:
I think it'd be nice if we moved these time helpers into their own contract file, e.g. `TimeHelpers`.